### PR TITLE
2017 column settings

### PR DESCRIPTION
**Pull request by aviveiros11**
#### Any background context you want to provide?
First PR, Woo!  This is addressing the column settings table sorting on the organizations page
#### What's this PR do?
Allows the user to now sort the Display name and Column name columns alphabetically. 
#### How should this be manually tested?
Click the header of either column to sort them and click again to return to the default state.  
#### What are the relevant tickets?
Solves issue [#2017](https://github.com/SEED-platform/seed/issues/2017)

### CODE REVIEW

File: seed/static/seed/partials/column_settings.html [hunk 1]
```diff
@@ -56,8 +56,12 @@ <h3 translate>MODIFYING_COLUMN_SETTINGS</h3>
                 <table id="column-table" class="table has_no_btm_margin">
                     <thead>
                         <tr>
-                            <th translate>Display Name</th>
-                            <th translate>Column Name</th>
+                            <th class="text-left" ng-click="toggle_display_name_order_sort()">
+                                <span>Display Name<i class="glyphicon glyphicon-sort"></i></span>
```
Comment (adrian-lara) [inline seed/static/seed/partials/column_settings.html]: I'm thinking we should make the icons all the same. I kind of like the `sort` icon you're using vs the `list` icon that existed before, but I don't have a strong opinion either way.

@axelstudios any thoughts?
Screen Shot 2020-01-24 at 12 38 29 PM

Comment (nllong) [reply seed/static/seed/partials/column_settings.html]: I agree with @adrian-lara -- that is the sort icon is a bit nicer!

Comment (axelstudios) [reply seed/static/seed/partials/column_settings.html]: Same - I like _sort_

**Comment by nllong** [comment]
Changes look good. Merging once tests pass.

**Pull request merged**